suite-common/suite-types/package.json

```diff
@@ -11,6 +11,7 @@
     },
     "dependencies": {
         "@reduxjs/toolkit": "1.8.3",
+        "@suite-common/invity": "workspace:*",
```

Comment:
probably not necessary?

Reply:
yes, 3a983db0f235c4cba5a8354683b4cdae0ad024eb